controller/src/main/java/io/pravega/controller/server/rest/RESTServer.java

```diff
@@ -67,7 +69,7 @@ protected void startUp() {
         long traceId = LoggerHelpers.traceEnterWithContext(log, this.objectId, "startUp");
         try {
             log.info("Starting REST server listening on port: {}", this.restServerConfig.getPort());
-            channel = NettyHttpContainerProvider.createServer(baseUri, resourceConfig, false);
+            httpServer = GrizzlyHttpServerFactory.createHttpServer(baseUri, resourceConfig, true);
```

Comment:
what is the last boolean param and just curious, Netty.createServer had a boolean for blocking where we chose false. 
And here Grizzly.createServer its true.